pkg/pods/pod.go

```diff
@@ -186,10 +186,6 @@ func (pod *Pod) buildRunitServices(launchables []hoist.Launchable) error {
 		for _, executable := range executables {
 			sbTemplate.AddEntry(executable.Service.Name, executable.Exec)
 		}
```

Comment:
by separate commit do you mean separate PR? Also this if statement is totally superfluous and will never execute if you read the code above it